pennylane/math/is_independent.py

```diff
@@ -203,7 +203,7 @@ def _get_random_args(args, interface, num, seed, bounds):
         if interface == "autograd":
 
             # Mark the arguments as trainable with Autograd
-            rnd_args = pnp.array(rnd_args, requires_grad=True)
+            rnd_args = [tuple(pnp.array(a, requires_grad=True) for a in arg) for arg in rnd_args]
```

Comment:
Also curious about the change here 🤔

Reply:
This was producing ragged arrays, raising the corresponding warning. It was very easy to fix and is not
related to the ragged array issues of QNodes, it was just not so nicely written by me back then, producing an array of a list of tuples of different sizes.